open-banking-accelerator/accelerators/ob-apim/repository/resources/apis/Accounts/account-info-swagger.yaml

```diff
@@ -0,0 +1,11046 @@
+swagger: '2.0'
+info:
+  title: AccountandTransactionAPI
+  description: Swagger for Account and Transaction API Specification
+  termsOfService: 'https://www.openbanking.org.uk/terms'
```

Comment:
Shall we remove Uk parts from these

Reply:
fixed with https://github.com/wso2/financial-open-banking/pull/1/commits/e070a5da1ffe1fe284282bcf48a213cdeedb7547